examples/webgl_loader_draco.html

```diff
@@ -26,7 +26,7 @@
 		// Configure and create Draco decoder.
 		var dracoLoader = new DRACOLoader();
 		dracoLoader.setDecoderPath( 'js/libs/draco/' );
-		dracoLoader.setDecoderConfig( { type: 'js' } );
+		//dracoLoader.setDecoderConfig( { type: 'js' } );
```

Comment:
Probably non intentional change?

Reply:
Oops. Totally unintentional 🤦 . Tried to revert, but made a mess. Uncommented in subsequent commit.